prime-router/src/main/kotlin/azure/observability/event/ReportStreamEventService.kt

```diff
@@ -296,28 +276,6 @@ class ReportStreamEventService(
         ).send()
     }
 
-    override fun sendSubmissionProcessingError(
```

Comment:
This didn't align with how the other reports were going to work and likely would have led to confusion as this was inverting parameters compared to other builders